.github/workflows/ci.yml

```diff
@@ -54,8 +54,8 @@ jobs:
         shell: bash -l {0}
         run: |
           python setup.py install
-          python setup.py test --addopts "--cache-clear -m 'not full_framework'"
-          python setup.py test --addopts "--cache-clear -m 'full_framework'"
+          pytest --cache-clear -m 'not full_framework'
```

Comment:
btw, this was the fix for avoiding the segfault. no idea where they came from...